tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh

```diff
@@ -24,7 +24,10 @@ apt-get install -y libhdf5-dev
 ln -s /usr/lib/powerpc64le-linux-gnu/libhdf5_serial.so /usr/lib/powerpc64le-linux-gnu/libhdf5.so
 ln -s /usr/lib/powerpc64le-linux-gnu/libhdf5_serial_hl.so /usr/lib/powerpc64le-linux-gnu/libhdf5_hl.so
 
-#pip is not installed yet, so use easy_install
+#Install pip here because h5py seg faults with python3 when installed via easy_install
```

Comment:
Is there a github issue, or bug we can link to here?

Reply:
There is no github issue , I will create one.

Reply:
I tried today to recreate this issue and was unable to, I'm not sure what was going on. I will remove this change.